salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -57,6 +58,12 @@ stringData:
       - '{% endraw %}https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana/login/generic_oauth{%
         raw %}'
       secret: 4lqK98NcsWG5qBRHJUqYM1
+    - id: metalk8s-cli
+      name: MetalK8s CLI
+      redirectURIs:
+      - http://localhost
+      - http://127.0.0.1
+      secret: jfakfka9i32242k4j2k42j42249kah
```

Comment:
Let's not add "virtual" clients if possible - these tests should work with `metalk8s-ui` I believe.